common/app/common/configuration.scala

```diff
@@ -590,7 +590,12 @@ class GuardianConfiguration extends Logging {
 
   object formstack {
     lazy val url = configuration.getMandatoryStringProperty("formstack.url")
-    lazy val oAuthToken = configuration.getMandatoryStringProperty("formstack.oauthToken")
+    object identity {
+      lazy val oauthToken = configuration.getMandatoryStringProperty("formstack.identity.oauthToken")
+    }
+    object editorial {
+      lazy val oauthToken = configuration.getMandatoryStringProperty("formstack.editorial.oauthToken")
+    }
```

Comment:
Is there a reason to keep these tokens in separate objects? Is there anything wrong with them being two values named `identityOAuthToken` and `editorialOAuthToken`?

Reply:
sounds good - I guess this simplifies the code